Source/Kesmai.Server/Game/Items/Equipment/Weapons/Weapon.cs

```diff
@@ -59,9 +59,22 @@ public override int CalculateBlockingBonus(ItemEntity item)
 		/// </summary>
 		public override double CalculateFumbleChance(MobileEntity entity)
 		{
-			// Skill Level = 3 =>  ((3 + 1)^2) * 10 = 160 => 1 / 160;
-			// Skill Level = 4 =>  ((4 + 1)^2) * 10 = 250 => 1 / 250;
-			return 1 / (10 * Math.Pow(entity.GetSkillLevel(Skill) + 1, 2));
+			var leftHand = LeftHand;
```

Comment:
leftHand and rightHand don't seem to add much to me, can probably ditch them and just use var leftWeapon = LeftHand as IWeapon;  not really a problem though